bom/pom.xml

```diff
@@ -162,6 +162,11 @@
          </dependency>
          <dependency>
             <groupId>${project.groupId}</groupId>
+            <artifactId>infinispan-commons-test</artifactId>
```

Comment:
+1000 for creating new commons test module. I heart was bleeding when I saw core's test jar included everywhere only because of the listeners :)